### ISSUE TITLE
Compilation fails

### ISSUE THREAD

**Issue by q1085909155** [ivenuss/flag-manager#2]
Dear @ivenuss The compile failure has not been fixed yet, and I have made sure that my compiler is up to date. Another problem is that whenever I restart the server, the plugin automatically refreshes the administrator in the backgroun
![QQ截图20210919235209](https://user-images.githubusercontent.com/69096633/133934188-ad850755-cf60-48c3-a9e0-756ce4f3df62.png)
![QQ截图20210919235255](https://user-images.githubusercontent.com/69096633/133934200-5a84b502-b4c6-4aa2-b5cb-4d7084172a12.png)
d

**Comment by ivenuss** [comment]
Hey @q1085909155,
does your include folder, where you compile this plugin includes `console.inc` file?

You can find more info here
[https://github.com/alliedmodders/sourcemod/blob/master/plugins/include/console.inc#L434-L439 ](url)

**Comment by q1085909155** [comment]
Yes, it is included, and my Sourcemod is also the latest version
